pyartifactory/objects/build.py

```diff
@@ -0,0 +1,191 @@
+from __future__ import annotations
+
+import logging
+from typing import Union
+
+import requests
+from requests import Response
+
+from pyartifactory.exception import ArtifactoryError, BuildNotFoundError
+from pyartifactory.models.build import (
+    BuildCreateRequest,
+    BuildDeleteRequest,
+    BuildDiffResponse,
+    BuildError,
+    BuildInfo,
+    BuildListResponse,
+    BuildPromotionRequest,
+    BuildPromotionResult,
+    BuildProperties,
+)
+from pyartifactory.objects.object import ArtifactoryObject
+
+logger = logging.getLogger("pyartifactory")
+
+
+class ArtifactoryBuild(ArtifactoryObject):
+    """Models an artifactory build."""
+
+    _uri = "build"
+
+    def get_build_info(
+        self,
+        build_name: str,
+        build_number: str,
+        properties: BuildProperties = BuildProperties(),
+    ) -> BuildInfo:
+        """
+        :param build_name: Build name to be retrieved
+        :param build_number: Build number to be retrieved
+        :param properties: Build properties model, for admitted values
+                           see https://jfrog.com/help/r/jfrog-rest-apis/build-info
+        :return: BuildInfo model object containing server response
+        """
+        build_name = build_name.strip("/")
+        build_number = build_number.strip("/")
```

Comment:
Actually, I wonder if we need to strip the values since the users are supposed to know the exact `build_name` and `build_number`. I think we can just remove those lines, if a user puts a slash or any unauthorized character it will just crash